JSDemos/Demos/DataGrid/GridSummaries/description.md

```diff
@@ -1 +1,12 @@
-The DataGrid component allows you to&nbsp;display data summaries within its container. This example illustrates the use of&nbsp;total summaries. The demo is&nbsp;configured using the **totalItems** array declared within the **summary** object. This array contains several objects that specify summary settings for each item.
\ No newline at end of file
+A total summary creates a synopsis for all data of a certain column. To configure this feature, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array with summary configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType)&mdash;an aggregate function that should be applied to this data.
```

Comment:
I probably wouldn't use the word "synopsis" to explain "total summaries". The phrase "total summaries" seems self-explanatory and definitely closer to the real meaning than "synopsis". So my suggestion is to simply go directly into the second sentence: 

 ```suggestion
To display total summaries, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array with configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType)&mdash;an aggregate function that should be applied to this data.
```